man/ostree.repo-config.xml

```diff
@@ -404,6 +404,17 @@ License along with this library. If not, see <https://www.gnu.org/licenses/>.
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>bls-append-except-default</varname></term>
+        <listitem><para>A semicolon seperated list of key-value pairs. For example:
+        <literal>bls-append-except-default=key1,value1;key2,value2</literal>. These key-value 
```

Comment:
Any reason to use `,` over e.g. `=` which would seem more obvious?

Reply:
I'm personally used to thinking of key-value pairs as two item tuples (comma-separated and ordered pair). Which is why my go to was to use `,`. However, I'm not bound by it. I'm OK with switching to `=`